client-side/src/tests/loader.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
tests should be located in client-side/test , and then in the same hierarchy as in the src folder.
as well, adding @testint-library (and jest, if needed) to package.json is missing